index.bs

```diff
@@ -755,8 +755,10 @@ spec.  It would be nice to unify these in the future.</p>
 To <dfn>clear cookies for host</dfn> given a [=host=] |host|, perform the
 following steps:
 
-1. Let |cookieList| be the set of cookies from the [=cookie store=] whose
-    domain attribute is a [=domain-match=] with |host|.
+1. Let |cookieList| be a set of cookies, initially empty.
+1. [=list/For each=] cookie |cookie| in the [=cookie store=], if either of the following is true, add |cookie| to |cookieList|:
+     - |cookie| is not partitioned, and |cookie|'s domain attribute is a [=domain-match=] with |host|; or
+     - |cookie| is partitioned, and |cookie|'s partition key is equal to |host|'s [=obtain a site|site=].
```

Comment:
This isn't worse than what was here before, so I don't object to merging it, but I'd really like to see "partitioned", "domain attribute", and "partition key" link to their definitions. I think "domain attribute" could link to https://datatracker.ietf.org/doc/html/rfc6265/#section-5.2.3 now. @johannhof / @dcthetall, is https://www.ietf.org/archive/id/draft-cutler-httpbis-partitioned-cookies-01.html the right citation for the partitioning aspects?

Reply:
I've added dfn links for those three terms as you suggested, but happy to update if there's a better doc to link to for CHIPS terms.